src/helm/benchmark/presentation/run_specs_dec2023.conf

```diff
@@ -0,0 +1,169 @@
+# HELM scenarios.
```

Comment:
If this is HELM lite, can we just write the config without priorities to improve legibility?

I.e. drop anything tier 3 or 4 in this file and upgrade all priority 2 to 1?

(Since nobody seems to use the tiers anyways, and I think the tiers inherited from HELM Nov '22 era don't really have a meaningful semantics anymore.)